### PR TITLE
Alert attendant if item is overdue and what payment is needed.

### PR DESCRIPTION
**Pull request by stevethomas19**
I talked with Harold about this one. The lab attendants know where to go to get payment. This just serves to alert them and to calculate the price that would be owed on an overdue item.

### CODE REVIEW

File: core/db/migration/migrate-db.sh [hunk 1]
```diff
@@ -16,6 +16,7 @@ echo "Implementing Migrations"
 
 psql -c "select addcol('public','users', 'last_login', 'timestamp', 'current_timestamp');"
 psql -c "select addcol('public','media', 'fine_amount', 'real', '0.50');"
+psql -c "select addcol('public','inventory', 'price', 'money', '1.00' );"
```
Comment (trevren11) [inline core/db/migration/migrate-db.sh]: @stevethomas19 I think we were going to use line 18 to make the prices for the items so we didn't have 3k+ fields to worry about updating. You should be able to use a join with media type and the item in the inventory and have a default price set if it doesn't have a type (might not be necessary if all items have a type)

**Comment by stevethomas19** [comment]
@trevren11 Thanks for the info on this one. I should have seen that before. The solution has been updated to use fine_amount. :)

**Comment by alexcdahl** [comment]
Looks good to me, but I don't know much about the fining system. I notice that in the mock data, some of the overdue items show a $0.00 when getting checked in. Does anybody who knows the system better know if this is normal?

**Comment by alexcdahl** [comment]
Just found out that we're good. I'm going to go ahead and merge.

**Pull request merged**